src/graph_notebook/magics/graph_magic.py

```diff
@@ -646,12 +649,33 @@ def load(self, line='', local_ns: dict = None):
             disabled=False,
         )
 
+        mode = widgets.Dropdown(
+            options=LOAD_JOB_MODES,
+            value=args.mode,
+            description='Mode:',
+            disabled=False
+        )
+
+        queue_request = widgets.Dropdown(
+            options=['TRUE', 'FALSE'],
+            value=str(args.queue_request).upper(),
+            description='Queue Request:',
+            disabled=False,
+        )
+
+        dependencies = widgets.Text(
+            value=args.dependencies,
+            placeholder='[\"load_A_id\", \"load_B_id\"]',
+            description='Dependencies:',
+            disabled=False
+        )
+
```

Comment:
We may want to use a `Textarea` here instead of just `Text` since it can give us new lines. Then it can be one dependency per line and we can build it into a list

https://ipywidgets.readthedocs.io/en/latest/examples/Widget%20List.html#Textarea

So then this would look something like
```
dependencies = widgets.Text(
    value="\n".join(args.dependencies),
    placeholder='load_A_id\nload_B_id',
    description='Dependencies:',
    disabled=False
)
```

Reply:
Done